site/app/controllers/UserProfileController.php

```diff
@@ -145,6 +145,29 @@ public function changePronouns() {
         }
     }
 
+    /**
+     * @Route("/user_profile/change_display_name_order", methods={"POST"})
+     * @return JsonResponse
+     */
+    public function changeDisplayNameOrder() {
+        $user = $this->core->getUser();
+        if (isset($_POST['display-name-order'])) {
+            $newPronouns = trim($_POST['display-name-order']);
```

Comment:
newPronouns being set to display-name-order is a bit confusing given the form is described as "You can edit the order in which your name is displayed here!"